### ISSUE TITLE
ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by bowencool** [pat-s/alfred-gitea#3]
<img width="671" alt="image" src="https://user-images.githubusercontent.com/20217146/161212569-c266e607-a70c-4781-a71c-da51dcb542bf.png">

**Comment by pat-s** [comment]
Does `pip3 install requests` help?

**Comment by bowencool** [comment]
> Does `pip3 install requests` help?

Yes, thanks.

**Comment by bowencool** [comment]
Hi, I am running into this problem again, but `pip3 install requests` is not working this time.

```
~
▶ pip3 install requests
Requirement already satisfied: requests in /opt/homebrew/lib/python3.10/site-packages (2.28.1)
Requirement already satisfied: certifi>=2017.4.17 in /opt/homebrew/lib/python3.10/site-packages (from requests) (2022.12.7)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /opt/homebrew/lib/python3.10/site-packages (from requests) (1.26.13)
Requirement already satisfied: charset-normalizer<3,>=2 in /opt/homebrew/lib/python3.10/site-packages (from requests) (2.1.1)
Requirement already satisfied: idna<4,>=2.5 in /opt/homebrew/lib/python3.10/site-packages (from requests) (3.4)

[notice] A new release of pip is available: 23.0 -> 23.0.1
[notice] To update, run: python3.10 -m pip install --upgrade pip

~
▶ which pip3
/opt/homebrew/bin/pip3
```

**Comment by pat-s** [comment]
You are using homebrew's python. Try the following

```
/usr/bin/python3 -m pip install requests
```
